lib/bundler/vendor/compact_index_client/lib/compact_index_client.rb

```diff
@@ -0,0 +1,71 @@
+require "pathname"
+require "bundler/worker"
```

Comment:
I don't know what this means... the client can't depend on Bundler::Worker? I didn't even write this code. :P